app/assets/stylesheets/finder_frontend.scss

```diff
@@ -410,3 +410,12 @@
   }
 }
 
+.finder-frontend-content {
+  .app-c-search-page-heading {
```

Comment:
Ideally we shouldn't have glue code like this for components but I can see why it was necessary. I think we should raise a separate PR to add some styling to the component to handle this.

Reply:
I agree, I've mention before we should extend the search component to allow it to be used as a page heading and that change would mean that we could remove this code.

Reply:
Is there a card or something that can be linked to from here? So people who come across this comment can see if this change to the search component has been made yet or not.